build.js

```diff
@@ -52,51 +43,45 @@ StyleDictionary.registerTransform({
   },
 });
 
+const customColorObjectFormatter = (dictionary, isJS) => {
```

Comment:
Do we want to add a default here? Or just require it to ensure that we get proper formatting

Reply:
i considered it but figured might be safer to just require it, 🍅 🍅 for me though so if you feel strongly either way i'm cool wit it